test/llvmpasses/havefma.ll

```diff
@@ -0,0 +1,24 @@
+; RUNx: opt --mtriple=`llvm-config --host-target` -enable-new-pm=1 --load-pass-plugin=libjulia-codegen%shlibext -passes='require<machine-module>,CPUFeatures' -S %s | FileCheck %s --check-prefixes=CHECK,CHECK-any
```

Comment:
`require<machine-module>` needs a LLVm patch right?

Reply:
yes, right now it this is disabled, with the `x`